stripe/src/main/java/com/stripe/android/view/CvcEditText.kt

```diff
@@ -64,6 +64,14 @@ class CvcEditText @JvmOverloads constructor(
                 completionCallback()
             }
         }
+
+        initParentOnFocusChangeListener { _, hasFocus ->
+            if (!hasFocus && unvalidatedCvc.normalized.length > 1) {
+                shouldShowError = unvalidatedCvc.validate(
```

Comment:
can this be `shouldShowError  = unvalidatedCvc.validate(cardBrand.maxCvcLength) == null`?